dltool/providers/fabricmc/fabricmc.go

```diff
@@ -0,0 +1,41 @@
+package fabricmc
+
+import (
+	"fmt"
+
+	"github.com/buger/jsonparser"
+	"github.com/go-resty/resty/v2"
+)
+
+var baseUrl = "https://meta.fabricmc.net/v2/versions"
+
+var client *resty.Client = resty.New()
+
+func getVersion(m Module) string {
+	// Make a API request for the given module
+	resp, err := client.R().Get(fmt.Sprintf("%s/%s", baseUrl, m))
+	if err == nil && resp.StatusCode() == 200 {
```

Comment:
I would probably negate and un-nest this if statement - just me being picky though.

Reply:
👀 I'm going to do this across the whole codebase in a separate PR. Thank you ^-^